src/effect.js

```diff
@@ -68,18 +68,19 @@ export class Effect {
 
 	deleteEffect() {
 		const targetIdx = this.target.effects.indexOf(this);
-		if (!this.target.effects[targetIdx]) {
-			console.warn('Failed to find effect on target.');
+		if (this.target.effects[targetIdx]) {
+			this.target.effects.splice(targetIdx, 1);
+		} else {
+			console.warn('Failed to find effect on target.', this);
 		}
-		this.target.effects.splice(targetIdx, 1);
 
 		const gameIdx = this.game.effects.indexOf(this);
-		if (!this.target.effects[gameIdx]) {
-			console.warn('Failed to find effect on game.');
+		if (this.game.effects[gameIdx]) {
```

Comment:
This fixed the Nutcase not applying "Tentacle Bush" to the Golden Wyrm after being executed (apply affect then instantly die).

Reply:
Hmm, I guess these is one of those things that can easily go both ways just fine as it would just be how the ability works.
But yeah, should probably consider those tentacles kinda like some slugs or serpents that just detach and live for a while.

Reply:
Makes sense. If we did want Tentacle Bush to not apply on a killing blow, we'd still fix this bug and change the ability.